src/ape/managers/chain.py

```diff
@@ -482,7 +482,14 @@ def restore(self, snapshot_id: Optional[SnapshotID] = None):
         self.provider.revert(snapshot_id)
         self.account_history.revert_to_block(self.blocks.height)
 
-    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> None:
```

Comment:
Yeah, this method should have had a docstring before, good time to add it!

Reply:
I figured, I'll add the docstring after I get back from the kids soccer practice